src/ApigeeEdgeSdkConnectorTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Drupal\apigee_m10n;
+
+/**
+ * Provides a trait with utility method to fetch Apigee's Edge sdk connector.
+ */
+trait ApigeeEdgeSdkConnectorTrait {
```

Comment:
Shouldn't this be in Apigee Edge module? This class does not have anything monetization specific.

Reply:
See https://github.com/apigee/apigee-m10n-drupal/pull/191#discussion_r371066752